### PR TITLE
Fix influxdb format configuration for kafka output

### LINKED ISSUE #1913: Kafka / Influxdb format: influxdb's ConcurrentWrites must be a positive number
Running a K6 load tests with following configuration:

```
#!/bin/bash
export K6_INFLUXDB_TAGS_AS_FIELDS=tag1,tag2
export K6_INFLUXDB_CONCURRENT_WRITES=5
export K6_KAFKA_BROKERS=localhost:29092,localhost:29093,localhost:29094
export K6_KAFKA_TOPIC=metrics
export K6_KAFKA_FORMAT=influxdb
export K6_KAFKA_PUSH_INTERVAL=1s
export K6_OUT=kafka

k6 run demo.js --verbose
```

Shows following error:

```
DEBU[0000] Starting executor run...                      duration=1m0s executor=constant-vus scenario=default type=constant-vus vus=100
ERRO[0001] Kafka: Couldn't format the samples            error="influxdb's ConcurrentWrites must be a positive number"
ERRO[0002] Kafka: Couldn't format the samples            error="influxdb's ConcurrentWrites must be a positive number"**
```

## Environment

- k6 version: v0.31.0 ((devel), go1.16, darwin/amd64)
- OS and version: macOS 11.2.1

## Expected Behavior

Send k6 metrics in InfluxDB format.

## Actual Behavior

Raise an error.

### PR DESCRIPTION
**Pull request by MStoykov**
fix #1913

### CODE REVIEW

**Review by na-- — DISMISSED** [review]
:disappointed:

**Review by imiric — APPROVED** [review]
You tested this on your machine? We need some integration tests for it, in another PR.

**Comment by na--** [comment]
Can you add a simple test for this please?

**Review by na-- — APPROVED** [review]

File: stats/kafka/config_test.go [hunk 1]
```diff
@@ -64,3 +69,55 @@ func TestConfigParseArg(t *testing.T) {
...
+			// hacks around env not actually being taken into account
+			os.Clearenv()
+			defer os.Clearenv()
+			for k, v := range testCase.env {
+				require.NoError(t, os.Setenv(k, v))
+			}
+
+			config, err := GetConsolidatedConfig(testCase.jsonRaw, testCase.env, testCase.arg)
+			if testCase.err != "" {
+				require.Error(t, err)
```
Comment (imiric) [inline stats/kafka/config_test.go:115]: There are no `err` cases to hit this, but OK :)

**Review by imiric — APPROVED** [review]

File: stats/kafka/config_test.go (hunk 1 above)
Comment (MStoykov) [reply stats/kafka/config_test.go:115]: I was really hoping that the "not positive concurrent writes" will break the config, but it breaks much later, and even with some refactoring I would've needed to also spin up kafka server to get the error ... which is *way* outside the scope of this PR or test

**Pull request merged**